app/master/build_request.py

```diff
@@ -22,6 +22,14 @@ class BuildRequest(object):
         "job_name": "clusterrunner_configured_job",
```

Comment:
Functionally, yeah. But I was thinking about what would display in all of the dashboards we have, and was worried that they will just break. What do you think I should do?